include/uv.h

```diff
@@ -1113,7 +1114,7 @@ struct uv_fs_s {
   uv_fs_cb cb;
   ssize_t result;
   void* ptr;
-  const char* path;
```

Comment:
undo this change, it creates too much churn. cast away the const where necessary (which should be a single place IIRC)